### PR TITLE
Feature/#97 update nodejs version

### PR DESCRIPTION
**Pull request by leoperetz**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM dwolla/alpine-node-make
+FROM mhart/alpine-node:8.9.4
```
Comment (rmothilal) [inline Dockerfile]: add the following under "FROM mhart/alpine-node:8.9.4"

RUN apk add --no-cache -t build-dependencies make gcc g++ python libtool autoconf automake \
    && cd $(npm root -g)/npm \
    && npm install fs-extra \
    && sed -i -e s/graceful-fs/fs-extra/ -e s/fs.rename/fs.move/ ./lib/utils/rename.js \
    && npm install -g node-gyp \
    && npm install -g sodium@1.2.3 --unsafe-perm \
    && npm install -g argon2@0.14.0 --unsafe-perm \
    && apk del build-dependencies

File: test/integration-runner.sh [hunk 2]
```diff
@@ -1,15 +1,16 @@
...
     exit 1
 fi
 
+POSTGRES_USER=${POSTGRES_USER:-postgres}
+POSTGRES_PASSWORD=${POSTGRES_PASSWORD:-postgres}
+POSTGRES_HOST=${HOST_IP:-localhost}
+INTG_TEST_CMD=${INTG_TEST_CMD:-tape \'test/integration/**/*.test.js\' | faucet}
+env_file=$1
+
 psql() {
```
Comment (mdebarros) [inline test/integration-runner.sh]: Lines 8-13: Should be above line 3.

**Comment by mdebarros** [comment]
I also do not see any Story linked to this PR?

File: Dockerfile (hunk 1 above)
Comment (rmothilal) [reply Dockerfile]: leo please delete the following lines:
 
&& npm install fs-extra 
&& sed -i -e s/graceful-fs/fs-extra/ -e s/fs.rename/fs.move/ ./lib/utils/rename.js

File: test/integration-runner.sh (hunk 2 above)
Comment (leoperetz) [reply test/integration-runner.sh]: ok, fixed

File: Dockerfile [hunk 3]
```diff
@@ -1,4 +1,13 @@
...
+
+RUN apk add --no-cache -t build-dependencies make gcc g++ python libtool autoconf automake \
+  && cd $(npm root -g)/npm \
+  && npm install fs-extra \
```
Comment (rmothilal) [inline Dockerfile]: please delete this line

File: Dockerfile [hunk 4]
```diff
@@ -1,4 +1,13 @@
...
+  && sed -i -e s/graceful-fs/fs-extra/ -e s/fs.rename/fs.move/ ./lib/utils/rename.js \
```
Comment (rmothilal) [inline Dockerfile]: please delete this line

File: Dockerfile [hunk 5]
```diff
@@ -1,4 +1,11 @@
-FROM dwolla/alpine-node-make
+FROM mhart/alpine-node:8.9.4
+
+RUN apk add --no-cache -t build-dependencies make gcc g++ python libtool autoconf automake \
```
Comment (rmothilal) [inline Dockerfile]: please remove "python" from this line and try.

**Comment by elnyry** [comment]
Another pull request raised with full changes, which is now merged.

**Pull request closed without merging**